Source/WebCore/bindings/js/JSDOMWindowBase.cpp

```diff
@@ -240,7 +239,7 @@ void JSDOMWindowBase::queueMicrotaskToEventLoop(JSGlobalObject& object, Ref<JSC:
     auto& eventLoop = thisObject.scriptExecutionContext()->eventLoop();
```

Comment:
Since `scriptExecutionContext()` is actually a function call (not an inlined one unless PGO made it so), how about caching it instead of calling twice?